airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/DbtCloudTransformationsCard.module.scss

```diff
@@ -63,20 +63,20 @@
   align-items: center;
 }
 
-.jobListItemInputGroup {
+.jobListItemIdFieldGroup {
   display: flex;
   justify-content: space-between;
   align-items: center;
+  flex-grow: 2;
 }
 
-.jobListItemInput {
+.jobListItemIdField {
   height: fit-content;
   margin-left: 1em;
-}
-
-.jobListItemInputLabel {
-  font-size: 11px;
-  font-weight: 500;
+  background-color: colors.$grey-50;
+  flex-grow: 2;
+  padding: variables.$spacing-sm;
+  border-radius: variables.$border-radius-sm;
```

Comment:
<img width="579" alt="image" src="https://user-images.githubusercontent.com/7550957/202412803-b452c0e9-0072-480c-b5c0-9ba09db681a7.png">

Styling of the icon / border radius looks a bit off? (icon too big, text too close to icon, border radius). Maybe I'm looking at an old [Figma design](https://www.figma.com/file/68CPMNSO3YLhSK72nhdOiM/DBT?node-id=507%3A15057&t=VpBmAYmftBNZaQ7x-0).

Reply:
Agreed, it could use a few small tweaks. The icon size at least is as designed, but the text is definitely a bit too close and now that it holds the job's display name instead of a generic "hey this is a dbt cloud job" title, it's probably a bit too small also. I'm going to merge this as-is for now, though, so Sophia can test it out while rewriting the documentation to reflect the updated UI.